features/support/env.el

```diff
@@ -20,6 +20,19 @@
 (Setup
  ;; Before anything has run
  (setq inhibit-message t) ;; Do not log out that minor mode is enabled/disabled (or other calls to (message) )
+
+ ;; This fixes an issue in emacs 25.1 where the debugger would be invoked
```

Comment:
Here's the bug report: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=24778 That code just replaces the function with the patched version. 

I should probably just upgrade my emacs to 25.2 to be honest...